lib/moonbeam.js

```diff
@@ -103,21 +103,24 @@ class Moonbeam {
 
     app.options('*', cmw)
 
+    app.get('/competition/leaderboard/vol', cmw, privRoutes.onCompetitionVolLbHttpRequest.bind(privRoutes))
```

Comment:
no need for auth here, the comeptition will be a public event, no?

Reply:
will remove, thank you